### DISCUSSION TITLE
Which to set up as the master

### DISCUSSION THREAD

**Discussion by indoorcat** [apohl79/audiogridder#649] [Help]
Hi everyone!
AudioGridder seems like a lifesaver!!
But I was wondering; I have 2 mac pro late 2013s, 1 with upgraded specs, and 1 with a pretty standard spec.
Which one do I make to be the master?

**Comment by CaBuffalo** [comment]
What do you mean by master, the server or the DAW?

On Sun, Jul 18, 2021 at 12:37 AM indoorcat ***@***.***> wrote:

> Hi everyone!
> AudioGridder seems like a lifesaver!!
> But I was wondering; I have 2 mac pro late 2013s, 1 with upgraded specs,
> and 1 with a pretty standard spec.
> Which one do I make to be the master?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/649>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6C2TGIOG4JXLTXBX2TTYKAELANCNFSM5AR4BLGQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by UncoveredPitch** [reply]
The server only has to run plug-ins so the more powerful computer should host the DAW as this has to run the audio(real-time) thread as well, which is the most CPU-intensive task. At least that's my understanding of the OP's question.

**Reply by indoorcat** [reply]
Thank you, so the server is better off having more single thread cpus rather than multi thread?

**Reply by UncoveredPitch** [reply]
Do you mean that single-core speed is more important than a higher number of cores at a lower speed? I do believe this is the case for audio in general. But how does this relate to master(DAW) vs server?  It's impossible to give any meaningful advice without knowing the specs of both your Macs and the kind of plug-ins you want to run.
